include/mqtt_config.h

```diff
@@ -59,6 +59,23 @@
 #define MQTT_WSS_DEFAULT_PORT       443
 #endif
 
+#ifdef CONFIG_MQTT_TASK_CORE_SELECTION_ENABLED
+#define MQTT_CORE_SELECTION_ENABLED CONFIG_MQTT_TASK_CORE_SELECTION_ENABLED
+#else
+#define MQTT_CORE_SELECTION_ENABLED 0
```

Comment:
You can use `#define MQTT_CORE_SELECTION_ENABLED CONFIG_MQTT_TASK_CORE_SELECTION_ENABLED` only